docs/source/references.bib

```diff
@@ -456,6 +478,19 @@ @Misc {pyabel-2022
   doi =		 {10.5281/zenodo.5888391}
 }
 
+@InProceedings {ronneberger-2015-unet,
+  author =	 {Olaf Ronneberger and Philipp Fischer and Thomas Brox},
+  title =	 {{U}-{N}et: Convolutional Networks for Biomedical Image Segmentation},
+  booktitle =	 {Proceedings of the 18th International Conference on Medical Image Computing and Computer-Assisted Intervention},
+  doi =		 {10.1007/978-3-319-24574-4_28},
+  volume = 9351,
+  pages =	 {234-241},
+  year =	 2015,
+  month =	 Oct,
+  address =	 {Munich, Germany},
+  publisher =	 {Springer},
```

Comment:
Suggest omitting Publisher for consistency with other entries.

Reply:
Omitted.